src/screens/transaction_details/utils.tsx

```diff
@@ -190,6 +190,100 @@ const getDataByType = (type: string) => {
       tagTheme: 'four',
       tagDisplay: 'txUnblockUserLabel',
     },
+
+    // ========================
+    // osmosis gamm/v1beta1
+    // ========================
+    '/osmosis.gamm.v1beta1.MsgCreatePool': {
+      model: MODELS.MsgCreatePool,
+      content: COMPONENTS.CreatePool,
+      tagTheme: 'four',
```

Comment:
each module should have a different tag color. We currently have ~8 so just go down the list

Reply:
I added two new colours and updated the code